server/src/main/java/org/tctalent/server/api/admin/CandidateBuilderSelector.java

```diff
@@ -354,19 +359,26 @@ private DtoBuilder examsDto() {
                 ;
     }
 
-    private DtoBuilder candidateAttachmentDto(DtoPropertyFilter userPropertyFilter) {
-        return new DtoBuilder()
+    private DtoBuilder candidateAttachmentDto(DtoPropertyFilter userPropertyFilter, DtoType type) {
+        final DtoBuilder builder = new DtoBuilder()
             .add("id")
             .add("type")
             .add("name")
             .add("location")
             .add("fileType")
             .add("migrated")
             .add("cv")
-            .add("createdBy", userDto(userPropertyFilter))
             .add("createdDate")
             .add("uploadType")
             .add("url")
             ;
+
+        if (DtoType.PREVIEW.equals(type)) {
```

Comment:
This should be:

```suggestion
        if (!DtoType.PREVIEW.equals(type)) {
```

Found this by comparing performance stats between my version and merged version - was a biggish discrepancy so I knew something was different. Found this!